### PR TITLE
huggingface-cli: add page

### PR DESCRIPTION
**Pull request by osbm**
Fixes: #12807 

- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):** 0.23.4

### CODE REVIEW

**Comment by osbm** [comment]
Ah sorry for those mistakes. Is it good now?

File: pages/common/huggingface-cli.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# huggingface-cli
+
+> Interact with Huggingface Hub.
```
Comment (spageektti) [inline pages/common/huggingface-cli.md:3]:
```suggestion
> Interact with Hugging Face Hub.
```
Like in their documentation.

File: pages/common/huggingface-cli.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+`huggingface-cli upload --repo-type {{repo_type}} {{repo_id}} {{local_path}} {{path_in_repo}}`
+
+- Scan cache to see downloaded repositories and their disk usage:
+
+`huggingface-cli scan-cache`
+
+- Delete cache interactively:
+
+`huggingface-cli delete-cache`
```
Comment (spageektti) [inline pages/common/huggingface-cli.md:37]:
```suggestion
> Login, manage local cache, download or upload files.
> More information: <https://huggingface.co/docs/huggingface_hub/guides/cli>.

- Login to Hugging Face Hub:

`huggingface-cli login`

- See the Username who logged in:

`huggingface-cli whoami`

- Log out:

`huggingface-cli logout`

- Print information about the environment:

`huggingface-cli env`

- Download number of files from an repository and print out the path (omit filenames to download entire repository):

`huggingface-cli download --repo-type {{repo_type}} {{repo_id}} {{filename1 filename2 ...}}`

- Upload an entire folder or a file to Hugging Face:

`huggingface-cli upload --repo-type {{repo_type}} {{repo_id}} {{local_path}} {{path_in_repo}}`

- Scan cache to see downloaded repositories and their disk usage:

`huggingface-cli scan-cache`

- Delete cache interactively:

`huggingface-cli delete-cache`
```

Comment (osbm) [reply pages/common/huggingface-cli.md]: Ah that's right. Fixing that immediately

File: pages/common/huggingface-cli.md (hunk 1 above)
Comment (osbm) [reply pages/common/huggingface-cli.md]: Fixed this with the latest commit

File: pages/common/huggingface-cli.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+> Interact with Hugging Face Hub.
+> Login, manage local cache, download or upload files.
+> More information: <https://huggingface.co/docs/huggingface_hub/guides/cli>.
+
+- Login to Hugging Face Hub:
+
+`huggingface-cli login`
+
+- See the Username who logged in:
```
Comment (gutjuri) [inline pages/common/huggingface-cli.md:11]:
```suggestion
- Display the name of the logged in user:
```

File: pages/common/huggingface-cli.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+- Log out:
+
+`huggingface-cli logout`
+
+- Print information about the environment:
+
+`huggingface-cli env`
+
+- Download number of files from an repository and print out the path (omit filenames to download entire repository):
```
Comment (gutjuri) [inline pages/common/huggingface-cli.md:23]:
```suggestion
- Download files from an repository and print out the path (omit filenames to download entire repository):
```

File: pages/common/huggingface-cli.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+
+- Upload an entire folder or a file to Hugging Face:
+
+`huggingface-cli upload --repo-type {{repo_type}} {{repo_id}} {{local_path}} {{path_in_repo}}`
+
+- Scan cache to see downloaded repositories and their disk usage:
+
+`huggingface-cli scan-cache`
+
+- Delete cache interactively:
```
Comment (gutjuri) [inline pages/common/huggingface-cli.md:35]:
```suggestion
- Delete the cache interactively:
```

**Review by gutjuri — COMMENTED** [review]
Thanks for your contribution. I have some suggestions.

**Comment by osbm** [comment]
Ah thanks for the suggestions. I dont know how I missed all these things. :(

Sorry for the poor quality PR. I will do better.

**Comment by spageektti** [comment]
> Ah thanks for the suggestions. I dont know how I missed all these things. :(
> 
> Sorry for the poor quality PR. I will do better.

No problem. Everyone made mistakes, even the most active contributors.

File: pages/common/huggingface-cli.md [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+
+`huggingface-cli env`
+
+- Download files from an repository and print out the path (omit filenames to download entire repository):
+
+`huggingface-cli download --repo-type {{repo_type}} {{repo_id}} {{filename1 filename2 ...}}`
+
+- Upload an entire folder or a file to Hugging Face:
+
+`huggingface-cli upload --repo-type {{repo_type}} {{repo_id}} {{local_path}} {{path_in_repo}}`
```
[the remark this answers is not in the document]
Comment (osbm) [reply pages/common/huggingface-cli.md:29]: oh this is a nice one

**Comment by gutjuri** [comment]
> Ah thanks for the suggestions. I dont know how I missed all these things. :(
> 
> Sorry for the poor quality PR. I will do better.

No worries dude, the suggestions were only a matter of making a good PR very good ;) Thanks again!

**Review by gutjuri — APPROVED** [review]

**Pull request merged**